manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "product-summary",
-  "version": "2.83.1",
+  "version": "2.83.2-beta.0",
```

Let me undo this since it was just for testing temporarily.
```suggestion
  "version": "2.83.1",
```